emperor/qiime_backports/parse.py

```diff
@@ -99,11 +96,11 @@ def parse_metadata_state_descriptions(state_string):
     result = {}
     state_string = state_string.strip()
     if state_string:
-        cols = map(strip, state_string.split(';'))
+        cols = list(map(strip, state_string.split(';')))
```

Comment:
Since this is iterated on in the next line, doesn't need to be changed to a list.